### ISSUE TITLE
Validation decimal fields

### ISSUE THREAD

**Issue by josejachuf** [gi0baro/weppy#203]
The validators gt, gte, lt and lte only work for integers?
I have a model with a decimal field and validation fails

if validation is
```
validation = {        
    'fdecimal': {'gte': 0}
 }
```

> def _gt(self, val1, val2, eq=False):
>     if eq:
>         return val1 >= val2
>     return val1 > val2

> TypeError: '>=' not supported between instances of 'str' and  'int'

if validation is
```
validation = {        
    'fdecimal': {'gte': 0}
 }
```

> def _gt(self, val1, val2, eq=False):
>     if eq:
>         return val1 >= val2
>     return val1 > val2

> TypeError: '>=' not supported between instances of 'str' and 'int'

if validation is
```
validation = {        
    'fdecimal': {'gte': decimal.Decimal('0')}
 }
```

> TypeError: '>=' not supported between instances of 'str' and 'decimal.Decimal'

**Comment by gi0baro** [comment]
@josejachuf this is fixed in the current master. I will probably release also a bugfix version of the 1.1 later today, 'cause I'm not ready to release 1.2 yet.

**Comment by gi0baro** [comment]
@josejachuf this is published also in 1.1.2
